ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/name/vch-creation-wizard-name.component.ts

```diff
@@ -87,16 +65,13 @@ export class VchCreationWizardNameComponent implements OnInit {
      */
     onCommit(): Observable<any> {
         // TODO: unit test this
-        return this.createWzService.checkVchNameUniqueness(
-            this.form.get('name').value)
+        return this.createWzService.checkVchNameUniqueness(this.form.get('name').value)
             .switchMap(isUnique => {
                 if (!isUnique) {
                     this.form.get('name').setErrors({
                         resourcePoolExists: true
                     });
-                    this.formErrMessage =
-                        'There is already a VirtualApp or ResourcePool that exists with the same name!';
-
+                    this.formErrMessage = 'There is already a VirtualApp or ResourcePool that exists with the same name';
```

Comment:
This is still going against our pattern of not having the error defined in the component. Can we have an error state set here and then checked in the template? Similar to our other errors.

Reply:
This is for an alert that is shown at the top of the current step when next button is clicked. Since we are showing this as a name field validation error, do we still need to show the alert?

Reply:
I wouldn't think so. Is there any UX direction for this? May want just check with Alister to see what he thinks.